lib/_007/Q.pm

```diff
@@ -129,10 +129,10 @@ class Q::Identifier does Q::Term {
 }
 
 class Q::Term::Regex does Q::Term {
-    has Q::Literal::Str $.contents;
+    has Val::Str $.contents;
```

Comment:
I'm not in a state where I can discuss it.

I changed it for the lisp/ast tests.